plugins/modules/system/open_iscsi.py

```diff
@@ -86,6 +86,14 @@
         - Whether the list of nodes in the persistent iSCSI database should be returned by the module.
         type: bool
         default: false
+    rescan:
+        description:
+        - Rescan an established session for discovering new targets.
+        - When I(target) is ommited, will rescan all sessions.
```

Comment:
```suggestion
        - When I(target) is omitted, will rescan all sessions.
```
According to Merriam-Webster the spelling is `omitted` (https://www.merriam-webster.com/dictionary/omitted).